elasticsearch/client_test.go

```diff
@@ -265,6 +270,58 @@ func TestAddDocument(t *testing.T) {
 	})
 }
 
+func TestBulkUpdate(t *testing.T) {
+	testSetup(t)
+	esIndex := "search_index"
+	esDestType := "docType"
+	esDestIndex := fmt.Sprintf("%s/%s", esIndex, esDestType)
+	bulk := make([]byte, 1)
+	esDestURL := "esDestURL"
+
+	Convey("Given that bulk update is a success", t, func() {
+		doFuncWithValidResponse := func(ctx context.Context, req *http.Request) (*http.Response, error) {
+			return successESResponse(), nil
+		}
+		httpCli := clientMock(doFuncWithValidResponse)
+		cli := elasticsearch.NewClientWithHTTPClientAndAwsSigner(testUrl, testSigner, true, httpCli)
```

Comment:
Signer not needed for these tests. Elasticsearch is mocked and hence no signing - this only adds confusion to the tests

Reply:
keeping it same as existing tests